src/modules/commander/commander_params.c

```diff
@@ -262,19 +262,30 @@ PARAM_DEFINE_INT32(COM_RC_ARM_HYST, 1000);
  * A non-zero, positive value specifies the time-out period in seconds after which the vehicle will be
  * automatically disarmed in case a landing situation has been detected during this period.
  *
- * The vehicle will also auto-disarm right after arming if it has not even flown, however the time
- * will always be 10 seconds such that the pilot has enough time to take off.
- *
  * A negative value means that automatic disarming triggered by landing detection is disabled.
  *
  * @group Commander
- * @min -1
- * @max 20
  * @unit s
  * @decimal 2
  */
+
 PARAM_DEFINE_FLOAT(COM_DISARM_LAND, 2.0f);
 
+/**
+ * Time-out for auto disarm if too slow to takeoff
+ *
+ * A non-zero, positive value specifies the time after arming, in seconds, within which the
+ * vehicle must take off (after which it will automatically disarm).
+ *
+ * A negative value means that automatic disarming triggered by a pre-takeoff timeout is disabled.
```

Comment:
@hamishwillee @catch-twenty-two I believe it should be "A zero or negative value means that ..."